### PR TITLE
Small Chnages

### PR DESCRIPTION
**Pull request by AnandAili**
review the changes and please merge it

### CODE REVIEW

File: src/main/java/smokefree/Application.java [hunk 1]
```diff
@@ -25,6 +25,7 @@
 public class Application {
 
     // Store the application context so classes can use programmatic dependency injection
+    // Hi there
```
Comment (AnandAiliCS008) [inline src/main/java/smokefree/Application.java]: please write proper comments

File: src/main/java/smokefree/Application.java [hunk 2]
```diff
@@ -51,6 +52,10 @@ public ObjectMapper objectMapper() {
         return new ObjectMapper();
     }
 
+    /*
+    * sdmfndsfnndslfn
```
Comment (AnandAiliCS008) [inline src/main/java/smokefree/Application.java]: better write javadoc

File: src/main/java/smokefree/AxonFactory.java [hunk 3]
```diff
@@ -117,4 +117,13 @@ public CommandGateway commandGateway(CommandBus commandBus) {
                 .dispatchInterceptors(newArrayList(new BeanValidationInterceptor<>()))
                 .build();
     }
+
+    @Singleton
+    public CommandGateway commandGatewayasdsad(CommandBus commandBus) {
```
Comment (AnandAiliCS008) [inline src/main/java/smokefree/AxonFactory.java]: method identifier is not proper

File: src/main/java/smokefree/Query.java [hunk 4]
```diff
@@ -55,6 +55,13 @@ public long totalVolunteers() {
                 .count();
     }
 
+    public long totalVolunteers1() {
```
Comment (AnandAiliCS008) [inline src/main/java/smokefree/Query.java]: Why duplicate methods

**Review by AnandAiliCS008 — CHANGES_REQUESTED** [review]

File: src/main/java/smokefree/Application.java (hunk 1 above)
Comment (AnandAili) [reply src/main/java/smokefree/Application.java]: done

File: src/main/java/smokefree/Application.java (hunk 2 above)
Comment (AnandAili) [reply src/main/java/smokefree/Application.java]: done

File: src/main/java/smokefree/AxonFactory.java (hunk 3 above)
Comment (AnandAili) [reply src/main/java/smokefree/AxonFactory.java]: done

File: src/main/java/smokefree/Query.java (hunk 4 above)
Comment (AnandAili) [reply src/main/java/smokefree/Query.java]: removed

**Review by AnandAiliCS008 — APPROVED** [review]

**Pull request merged**